docs/settings/_font-settings.md

```diff
@@ -0,0 +1,11 @@
+---
+collection: settings
+title: Assets settings
+---
+
+Setting  | Default Value
+ ------------- | -------------
+$font-assets-path   | 'https://assets.ubuntu.com/sites/ubuntu/latest/u/'
+$font-base-family   | 'Ubuntu, Arial, "libra sans", sans-serif'
+$font-base-size   | 1rem
```

Comment:
This will set the body font size to `1rem` we should set the body font size as a px value and use rem from there